.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
-before_install: "npm install -g npm"
```

Comment:
Sliding a Travis CI change in and covering it up ay? ;) Those who forget history are bound to repeat it.